tests/ui/ferrocene/compiler-arguments/print/print_ok-option_print_different-option.stdout

```diff
@@ -168,7 +168,6 @@ thumbv8m.main-none-eabi
 thumbv8m.main-none-eabihf
 wasm32-unknown-emscripten
 wasm32-unknown-unknown
-wasm32-wasi
```

Comment:
Shouldn't `wasm32-wasip1` exist since it will replace this? https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html

Reply:
I wasn't aware of this, thanks! I'll replace `wasm32-wasi` by this instead and check if the `ui` tests still pass.

Reply:
we should in future regenerate this file with `compare-output-lines-by-subset` test annotation removed, because we have a lot more targets than displayed here